core/tests/test_utils.py

```diff
@@ -1153,11 +1153,6 @@ def save_new_exp_with_states_schema_v0(self, exp_id, user_id, title):
         )
         exp_summary_model.put()
 
-        # Note: Also save state id mappping model for new exploration. If not
```

Comment:
Maybe this should have been a red flag before :)

Reply:
True... I still don't understand how we missed out entries, but it did happen!